examples/app.py

```diff
@@ -99,8 +75,6 @@
     OAISERVER_RECORD_INDEX='_all',
     OAISERVER_ID_PREFIX='oai:localhost:recid/',
     SECRET_KEY='CHANGE_ME',
-    SQLALCHEMY_DATABASE_URI=os.getenv('SQLALCHEMY_DATABASE_URI',
```

Comment:
I have seen this in other PRs too. Please keep it so it's easy to change the database. Thanks